### ISSUE TITLE
returnAfterFirst(true) not working for images

### ISSUE THREAD

**Issue by mdumrauf** [esafirm/android-image-picker#60]
Version: 1.7.3 and starting the picker with:

```java
ImagePicker.create(SomeFragment.this)
                .returnAfterFirst(true)
                .single()
                .imageTitle("Tap an image")
                .showCamera(true)
                .imageDirectory("Camara")
                .start(REQUEST_CODE_PICKER);
```

The picker starts, it doesn't display the "DONE" button and enables single select mode. However, it is not returning after first tap. It is working for camera though.

Related #12

**Comment by esafirm** [comment]
@mdumrauf Fixed in `1.7.4` kindly check in here
https://github.com/esafirm/android-image-picker/releases/tag/1.7.4-beta-1

Cheers!

**Comment by mdumrauf** [comment]
Thanks! Working like charm @esafirm.
